examples/connext_dds/remote_procedure_call/c++11/Inventory_service.cxx

```diff
@@ -126,7 +112,38 @@ int main(int argc, char *argv[])
     ::InventoryServiceService service(service_impl, server, params);
 
     std::cout << "InventoryService running... " << std::endl;
-    server.run();
+    server.run(std::chrono::seconds(service_timeout));
```

Comment:
For us to test the examples they have to run in a given time, they cannot run forever